app/src/main/res/values-de/arrays-de.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string-array name="SendToPebbleChoice">
+        <item>Standard Watchface (alt)</item>
+        <item>Farbtrend Watchface</item>
+        <item>Pebble Classic Trend Watchface</item>
+        <item>Pebble Trend Clay Version (Test)</item>
+    </string-array>
+
+
+    <string-array name="PebbleTrendPeriods">
+        <item>1 Stunde</item>
+        <item>2 Stunden</item>
+        <item>3 Stunden</item>
+        <item>4 Stunden</item>
+    </string-array>
+
+    <string-array name="UpdateChannelDetail">
+        <item>Stable - bewährte und getestete Versionen</item>
+        <item>Beta - neue Funktionen, aber nicht gänzlich getestet</item>
+        <item>Alpha - Versionen zum frühen Testen</item>
+    </string-array>
+
+    <string-array name="UpdateChannelDetailE">
+        <item>Stable - bewährte und getestete Versionen</item>
+        <item>Beta - neue Funktionen, aber nicht gänzlich getestet</item>
+        <item>Alpha - Versionen zum frühen Testen</item>
+        <item>Nightly - ungetesteter Rohcode - nur Experten!</item>
+    </string-array>
+
+    <string-array name="TreatmentCalibrationChoiceDetail">
```

Comment:
`TreatmentCalibrationChoiceDetail` was recently updated to include 4 entries in total. I think the German translation needs to have the same. I am not sure what happens if there is a mismatch in the number of entries.